### PR TITLE
Force play timezone

### PR DESCRIPTION
**Pull request by hugovalente-pm**
##### Summary
Documentation changes to include the time control and timezone selector

##### Component Name
Documentation

##### Test Plan

Merge and validate the docs

##### Additional Information
N/A

### CODE REVIEW

File: docs/dashboard/visualization-date-and-time-controls.mdx [hunk 1]
```diff
@@ -38,16 +42,49 @@ Click any of the following options in the **Quick Selector** to choose a commonl
...
+of these changes
+* You want to have Netdata on the background, example displayed on a TV, to constantly see metrics through dashboards or to watch the alert 
+status
+
+![The time control with Play, Pause and 
+Force Play](https://user-images.githubusercontent.com/82235632/129206460-03c47d0d-1a5b-428a-b972-473718b74bdb.png)
+
+## Timezone selector
+
+With the timezone select you have the ability to change the timezone both in Agent and on Netdata Cloud. More often than not are
```
Comment (manos-saratsis) [inline docs/dashboard/visualization-date-and-time-controls.mdx:77]: @hugovalente-pm the timezone on the agent will take a bit longer

**Review by manos-saratsis — APPROVED** [review]
Approved but timezone on agent will be done later. You can keep it though as the functionality is already there through agent settings

File: docs/dashboard/visualization-date-and-time-controls.mdx (hunk 1 above)
Comment (hugovalente-pm) [reply docs/dashboard/visualization-date-and-time-controls.mdx:77]: Will rectify this and update the PR, thanks

File: docs/dashboard/visualization-date-and-time-controls.mdx [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 ---
 
-# Pick timeframes to visualize
+# Visualization date and time controls
+
+## Date and time selector
+
+### Pick timeframes to visualize
 
 While [panning through time and zooming in/out](/docs/dashboard/interact-charts.mdx) from charts is helpful when
```
Comment (jlbriston) [inline docs/dashboard/visualization-date-and-time-controls.mdx:14]:
```suggestion
While [panning through time and zooming in/out](/docs/dashboard/interact-charts.mdx) from charts it is helpful
```

File: docs/dashboard/visualization-date-and-time-controls.mdx [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
 you're looking a recent history, or want to do granular troubleshooting, what if you want to see metrics from 6 hours
```
Comment (jlbriston) [inline docs/dashboard/visualization-date-and-time-controls.mdx:15]:
```suggestion
for looking at recent history, or for granular troubleshooting, if you want to see metrics for the previous 6 hours
```

File: docs/dashboard/visualization-date-and-time-controls.mdx [hunk 4]
```diff
@@ -38,16 +42,49 @@ Click any of the following options in the **Quick Selector** to choose a commonl
...
+## Time controls
+
+The time control provides you the following options: **Play**, **Pause** and **Force Play**.
+* **Play** - the content of the page will be automatically refreshed while this is in the foreground
+* **Pause** - the content of the page isn't refreshed due to a manual request to pause it or, for example, when your investigating data on a 
+chart (cursor is on top of a chart)
+* **Force Play** - the content of the page will be automatically refreshed even if this is in the background
+
+With this, we aim to bring more clarity and allow you to distinguish if the content you are looking at is live or historical and also allow you
+ to always refresh the content of the page when the tabs are on the background.
```
Comment (jlbriston) [inline docs/dashboard/visualization-date-and-time-controls.mdx:64]: >
```suggestion
 to always refresh the content of the page when the tabs are in the background. 
```

File: docs/dashboard/visualization-date-and-time-controls.mdx [hunk 5]
```diff
@@ -38,16 +42,49 @@ Click any of the following options in the **Quick Selector** to choose a commonl
...
+* **Play** - the content of the page will be automatically refreshed while this is in the foreground
+* **Pause** - the content of the page isn't refreshed due to a manual request to pause it or, for example, when your investigating data on a 
+chart (cursor is on top of a chart)
+* **Force Play** - the content of the page will be automatically refreshed even if this is in the background
+
+With this, we aim to bring more clarity and allow you to distinguish if the content you are looking at is live or historical and also allow you
+ to always refresh the content of the page when the tabs are on the background. 
+
+Main use cases for **Force Play**:
+* You use a terminal or deployment tools to do changes in your infra and want to see immediately, Netdata is on the background, the impact
```
Comment (jlbriston) [inline docs/dashboard/visualization-date-and-time-controls.mdx:67]:
```suggestion
* You use a terminal or deployment tools to do changes in your infra and want to see immediately, Netdata is in the background, displaying the impact 
```

File: docs/dashboard/visualization-date-and-time-controls.mdx [hunk 6]
```diff
@@ -38,16 +42,49 @@ Click any of the following options in the **Quick Selector** to choose a commonl
...
+of these changes
+* You want to have Netdata on the background, example displayed on a TV, to constantly see metrics through dashboards or to watch the alert 
+status
+
+![The time control with Play, Pause and 
+Force Play](https://user-images.githubusercontent.com/82235632/129206460-03c47d0d-1a5b-428a-b972-473718b74bdb.png)
+
+## Timezone selector
+
+With the timezone select you have the ability to change the timezone on Netdata Cloud. More often than not are
```
Comment (jlbriston) [inline docs/dashboard/visualization-date-and-time-controls.mdx:77]:
```suggestion
With the timezone select, you have the ability to change the timezone on Netdata Cloud. More often than not are, there's a distribution in different timezones, requiring association. 
```

File: docs/dashboard/visualization-date-and-time-controls.mdx [hunk 7]
```diff
@@ -38,16 +42,49 @@ Click any of the following options in the **Quick Selector** to choose a commonl
...
+* You want to have Netdata on the background, example displayed on a TV, to constantly see metrics through dashboards or to watch the alert 
+status
+
+![The time control with Play, Pause and 
+Force Play](https://user-images.githubusercontent.com/82235632/129206460-03c47d0d-1a5b-428a-b972-473718b74bdb.png)
+
+## Timezone selector
+
+With the timezone select you have the ability to change the timezone on Netdata Cloud. More often than not are 
+distributed in different timezones and they need to collaborate.
```
Comment (jlbriston) [inline docs/dashboard/visualization-date-and-time-controls.mdx:78]:
```suggestion

```

File: docs/dashboard/visualization-date-and-time-controls.mdx [hunk 8]
```diff
@@ -38,16 +42,49 @@ Click any of the following options in the **Quick Selector** to choose a commonl
...
+
+Our goal is to make it easier for you and your teams to troubleshoot on your timezone preference and communicate easily
```
Comment (jlbriston) [inline docs/dashboard/visualization-date-and-time-controls.mdx:80]:
```suggestion
Our goal is to make it easier for you and your teams to troubleshoot based on your timezone preference and communicate easily 
```

File: docs/dashboard/visualization-date-and-time-controls.mdx [hunk 9]
```diff
@@ -38,16 +42,49 @@ Click any of the following options in the **Quick Selector** to choose a commonl
...
+![The time control with Play, Pause and 
+Force Play](https://user-images.githubusercontent.com/82235632/129206460-03c47d0d-1a5b-428a-b972-473718b74bdb.png)
+
+## Timezone selector
+
+With the timezone select you have the ability to change the timezone on Netdata Cloud. More often than not are 
+distributed in different timezones and they need to collaborate.
+
+Our goal is to make it easier for you and your teams to troubleshoot on your timezone preference and communicate easily 
+timezones and timeframes without need to be concerned about the specificity of them.
```
Comment (jlbriston) [inline docs/dashboard/visualization-date-and-time-controls.mdx:81]:
```suggestion
with varying timezones and timeframes without the need to be concerned about their specificity.
```

**Review by jlbriston — DISMISSED** [review]

File: docs/dashboard/visualization-date-and-time-controls.mdx (hunk 3 above)
Comment (hugovalente-pm) [reply docs/dashboard/visualization-date-and-time-controls.mdx:15]: it was missing a when, don't now how it gotten taken out, so it should read:

"While panning through time and zooming in/out from charts is helpful when you're looking a recent history..." I think that was why you suggested the ".. for looking"

File: docs/dashboard/visualization-date-and-time-controls.mdx (hunk 6 above)
Comment (hugovalente-pm) [reply docs/dashboard/visualization-date-and-time-controls.mdx:77]: here it was missing "teams", so it should read "More often than not team are distributed in different timezones and they need to collaborate."

**Review by manos-saratsis — APPROVED** [review]

**Review by jlbriston — APPROVED** [review]

**Pull request merged**